manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -828,6 +829,7 @@ data:
         traffic.sidecar.istio.io/excludeOutboundPorts: "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/excludeOutboundPorts` .Values.global.proxy.excludeOutboundPorts }}"
       {{- end }}
         traffic.sidecar.istio.io/kubevirtInterfaces: "{{ index .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces` }}"
+      revision: {{ valueOrDefault .Values.revision "" }}
```

Comment:
Is this field named revision as the idea would be users would have two different versions of istiod running?

Reply:
Currently the charts leave the `revision` blank by default and for the case we have only one control plane.

I feel this is confusing; I would prefer "revision" to default to `global` or to work for example "docker" which assigns a default name such as `hyperactive-marx`.

We could suppress `revision` label injection if there is no revision.  I did it this way because I think it is simpler, and also because I was thinking of the case where the user installs 1.6.0 without a revision but then installs 1.6.1 with revision `canary`.  It would be confusing to see the label on some pods but not others.